src/app/lib/config/services/korean.js

```diff
@@ -154,6 +154,7 @@ export const service = {
     },
     radioSchedule: {
       hasRadioSchedule: true,
+      header: 'BBC News Radio',
```

Comment:
Why do some services have `frequenciesPageLabel ` and `header` and some don't?

Reply:
@OlgaLyubin Only services that have `frequenciesPageUrl` should have a corresponding `frequenciesPageLabel`